tests/test_change_debt_with_profit.py

```diff
@@ -60,7 +60,7 @@ def test_change_debt_with_profit(
             new_params["totalGain"] - prev_params["totalGain"], donation, abs_tol=10
         )
     else:
-        assert new_params["totalGain"] - prev_params["totalGain"] > donation
+        assert new_params["totalGain"] - prev_params["totalGain"] >= donation
```

Comment:
changed to '>=' to include the scenario in which the gauge has no CRV to distribute